pom.xml

```diff
@@ -931,7 +931,7 @@
       <dependency>
         <groupId>com.spotify</groupId>
         <artifactId>docker-client</artifactId>
-        <version>5.0.2</version>
+        <version>8.14.1</version>
```

Comment:
Hi, @gaborgsomogyi . They are using `shaded` version, aren't they? Could you change like them to follow correctly (if we decide to follow them)?

Reply:
Tried it already but getting this:
```
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.version' for com.spotify:docker-client:jar:shaded is missing. @ line 49, column 17
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.apache.spark:spark-docker-integration-tests_2.12:3.1.0-SNAPSHOT (/Users/gaborsomogyi/spark/external/docker-integration-tests/pom.xml) has 1 error
[ERROR]     'dependencies.dependency.version' for com.spotify:docker-client:jar:shaded is missing. @ line 49, column 17
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
```
Why do we need shaded version?

Reply:
This is effectively a version upgrade in `dockeer-integration-tests/pom.xml`. We don't know what it brings. We have a history. Please see SPARK-29546.

Reply:
BTW, `com.spotify:docker-client:jar:shaded` is a correct message? `shaded` is a classifier, not a version.

Reply:
Had some problem with my repo setup but resolved and re-tested. Pushing it soon...

Reply:
> BTW, com.spotify:docker-client:jar:shaded is a correct message?

That was the flag to me as well that I must have local problem...

Reply:
Do we have any concerns back porting this into 3.0? any downside?